package.json

```diff
@@ -47,10 +47,10 @@
     "react": "^15.2.0",
     "react-native": "^0.30.0",
     "react-relay": "https://github.com/alloy/relay/releases/download/v0.9.3/react-relay-0.9.3.tgz",
-    "remove-markdown": "0.1.0",
-    "apphub": "^0.5.1"
+    "remove-markdown": "0.1.0"
   },
   "devDependencies": {
+    "apphub": "^0.5.1",
```

Comment:
Yeah, it's all in the pod, isn't needed in the module itself - see the lack of JS changes in https://github.com/artsy/emission/pull/263